README.md

```diff
@@ -91,6 +91,7 @@ variables (or a combination of the two):
 | `-max-duration` | `MAX_DURATION` | Maximum duration a response may take | 10s |
 | `-port` | `PORT` | Port to listen on | 8080 |
 | `-use-real-hostname` | `USE_REAL_HOSTNAME` | Expose real hostname as reported by os.Hostname() in the /hostname endpoint | false |
+| `-exclude-headers` | `EXCLUDE_HEADERS` | Drop platform-specific headers. Comma-separated list of headers key to drop, supporting wildcard matching. For example: `"foo,bar,x-fc-*"` | - |
```

Comment:
Thanks for thinking to update the README!  Would you mind also adding a note under the **Production considerations** section mentioning this new config option as a way to prevent leaking sensitive headers?

Reply:
Also, per my suggestion below about only allowing a single suffix wildcard, I might suggest a tweak here:

```suggestion
| `-exclude-headers` | `EXCLUDE_HEADERS` | Drop platform-specific headers. Comma-separated list of headers key to drop, supporting wildcard suffix matching. For example: `"foo,bar,x-fc-*"` | - |
```